cypress-tests/cypress/support/utils/manageUsers.js

```diff
@@ -401,3 +401,37 @@ export const inviteUserWithUserRole = (firstName, email, role) => {
   cy.get(commonSelectors.acceptInviteButton).click();
   cy.get(commonSelectors.homePageLogo, { timeout: 10000 }).should("be.visible");
 };
+
+export const inviteUserWithUserRoleAndMetadata = (firstName, email, role) => {
```

Comment:
Please reuse the existing function instead of creating a new one. If minor adjustments are needed for this use case, consider adding parameters to the existing function to handle the variation.